k-distribution/src/main/scripts/bin/kx

```diff
@@ -11,26 +11,30 @@ depth=
 params=()
 allPath=true
 dir=.
-cmd=
+cmd=kore-exec
```

Comment:
This variable should be named a little better, because it appears it's only for the Haskell backend? Or at least, it's only for when doing search or prove?

I would say it should be `haskellCmd`, instead of just `cmd`. If later we change LLVM backend to handle things like search or prove, then we can change the variable name.